examples/hid_mouse/src/main.c

```diff
@@ -172,23 +172,25 @@ int read_from_flash(void *db, const int max_size)
 #if (INGCHIPS_FAMILY == INGCHIPS_FAMILY_916)
 static void QDEC_PclkCfg(void)
 {
+    if ((APB_SYSCTRL->QdecCfg >> 15) & 1)
+        return;
     uint32_t hclk = SYSCTRL_GetHClk();
-    uint32_t qdecClk = SYSCTRL_GetClk(SYSCTRL_ITEM_APB_QDEC);
-    uint8_t div = hclk / qdecClk;
-    if (hclk % qdecClk)
+    uint32_t slowClk = SYSCTRL_GetSlowClk();
+    uint8_t div = hclk / slowClk;
+    if (hclk % slowClk)
```

Comment:
`SYSCTRL_GetClk(SYSCTRL_ITEM_APB_QDEC)` is still applicative in this case.

Reply:
Is these wrong?